sources/packages/api/src/utilities/student-utils.ts

```diff
@@ -0,0 +1,18 @@
+import { StudentPDStatus } from "../types/pdStatus";
+import { Student } from "../database/entities";
+
+/**
+ * Determines the permanent disablility status from Student Entity.
+ * @param student
+ * @returns PDStatus enum
+ */
+export const determinePDStatus = (student: Student): StudentPDStatus => {
+  if (student.studentPDVerified) {
+    return StudentPDStatus.Yes;
+  } else if (student.studentPDSentAt) {
+    return student.studentPDUpdateAt
+      ? StudentPDStatus.No
```

Comment:
I would prefer having the "Yes/No" status defined exclusively on "student.studentPDVerified" and, if "student.studentPDVerified" is not defined, check the others conditions.